### PR TITLE
Add ScriptingHelpers

### PR DESCRIPTION
**Pull request by Reprevise**
I added the site _ScriptingHelpers_ and it's icon in the _development_ section.

### CODE REVIEW

File: _data/developer.yml [hunk 1]
```diff
@@ -483,6 +483,11 @@ websites:
       twitter: saucelabs
       facebook: saucelabs
       email_address: support@saucelabs.com
+      
+    - name: Scripting Helpers
+      url: https://scriptinghelpers.org/
+      img: scriptinghelpers.png
+      tfa: Yes
```
Comment (kmpoppe) [inline _data/developer.yml]:
```suggestion
      tfa: Yes
      software: Yes
```

**Review by kmpoppe — CHANGES_REQUESTED** [review]
Hi @EcoGames and thank you for your contribution. I've added a single line comment that I'd ask you to commit because you didn't incorporate the 2fa-method in your PR.

**Comment by kmpoppe** [comment]
Also, would you be able to add some documentation or screenshots of the enabling process?

**Comment by Reprevise** [comment]
Where would you like the screenshots?

**Comment by kmpoppe** [comment]
You can just add them into a comment using drag & drop. If possible, add a describing line per step you take.

**Comment by Reprevise** [comment]
1. From the [main page](https://scriptinghelpers.org/), hover over your account name and click on "Settings".
image
2. From the "Settings" page, click on the Security tab.
image
3. From the "Security" tab, click on "Enable" under the "Two-Factor Auth" section.
image
4. Scan the QR code provided on-screen using any third-party authenticator app. (It doesn't have to be Google Authenticator!) Then, type in the code provided by your third-party app after you scan the QR code.
image
5. From there, when you go back to the "Security" tab, you will see that 2FA is enabled.
image

**Review by kmpoppe — APPROVED** [review]

**Review by kmpoppe — DISMISSED** [review]

**Comment by Jawshy** [comment]
Hi @Reprevise. Which [site criteria](https://github.com/2factorauth/twofactorauth/blob/master/CONTRIBUTING.md#site-criteria) does Scripting Helpers meet?

**Comment by Reprevise** [comment]
ScriptingHelpers is essentially a StackOverflow for Lua scripting on Roblox. It deals with personal information.

**Review by Jawshy — APPROVED** [review]

File: _data/developer.yml [hunk 2]
```diff
@@ -479,6 +479,12 @@ websites:
     facebook: saucelabs
     email_address: support@saucelabs.com
 
+  - name: Scripting Helpers
+    url: https://scriptinghelpers.org/
+    img: scriptinghelpers.png
+    tfa:
+      - software
```
Comment (kmpoppe) [inline _data/developer.yml]:
```suggestion
      - totp
```
That would be totp now :)

**Review by kmpoppe — CHANGES_REQUESTED** [review]
See inline comments, please :-)

**Review by kmpoppe — APPROVED** [review]

**Pull request merged**